android/runtime/v8/src/native/TypeConverter.cpp

```diff
@@ -239,7 +240,7 @@ jobject TypeConverter::jsObjectToJavaFunction(Isolate* isolate, JNIEnv *env, Loc
 {
 	Local<Function> func = jsObject.As<Function>();
 	Persistent<Function, CopyablePersistentTraits<Function>> jsFunction(isolate, func);
-	jsFunction.MarkIndependent();
+	// jsFunction.MarkIndependent(); // Method has been removed!
```

Comment:
Do we need to do anything here?

If this is where we box anonymous JS callbacks, then we shouldn't make it a weak reference since it might get garbage collected before invoked on the Java side.

Reply:
I think this is okay, it looks like we store a persistent reference to the JS function so it can't be garbage collected and release the reference when the Java function is destroyed `Java_org_appcelerator_kroll_runtime_v8_V8Function_nativeRelease`